airbyte-integrations/connectors/source-salesforce/pyproject.toml

```diff
@@ -18,7 +18,7 @@ include = "source_salesforce"
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
 pandas = "2.2.1"
-airbyte-cdk = "^0"
+airbyte-cdk = "0.83.1"
```

Comment:
Why upper bound this to 0.83.1?

Reply:
Same question for the rest of the PR's like this.